polly/lib/CodeGen/CodeGeneration.cpp

```diff
@@ -233,6 +238,26 @@ static bool generateCode(Scop &S, IslAstInfo &AI, LoopInfo &LI,
   NodeBuilder.allocateNewArrays(StartExitBlocks);
   Annotator.buildAliasScopes(S);
 
+  // The code below annotates the "llvm.loop.vectorize.enable" to false
+  // for the code flow taken when RTCs fail. Because we don't want the
+  // Loop Vectorizer to come in later and vectorize the original fall back
+  // loop when 'polly-annotate-metadata-vectorize' is passed.
+  if (PollyVectorizeMetadata) {
+    for (Loop *L : LI.getLoopsInPreorder()) {
+      if (S.contains(L)) {
```

Comment:
```suggestion
      if (!S.contains(L)) 
        continue;
```
[Coding standard](https://llvm.org/docs/CodingStandards.html#use-early-exits-and-continue-to-simplify-code)